cadvisor.yaml

```
@@ -30,6 +30,14 @@ pipeline:
 
   - uses: strip
 
+subpackages:
+  - name: ${{package.name}}-compat
+    description: Compatibility package for cadvisor
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.contextdir}}"/run
+          ln -sf /var/run "${{targets.contextdir}}"/run
```

```suggestion
          mkdir -p "${{targets.contextdir}}"/var/run
          ln -sf ../run "${{targets.contextdir}}"/var/run
```
Usually the "right way" would be to symlink `/run` to `../run` - does this break anything?

```shell
docker run --rm -it quay.io/fedora/fedora:latest ls -lah /var/run
```
```
lrwxrwxrwx 1 root root 6 Jun 10 06:48 /var/run -> ../run

```